src/main/java/frc/team4909/robot/RobotConstants.java

```diff
@@ -4,27 +4,25 @@
 
 public class RobotConstants {
   /* Operator Input */
-  public static final int driverGamepadPort = 0;
   public static final double driverGamepadSensitivity = 0.6; // Value selected from 2018 code
-  public static final int driverGamepadDeadzone = 0; // Value selected from 2018 code
+  public static final double driverGamepadDeadzone = 0.05; // Value selected from 2018 code
 
-  public static final int manipulatorGamepadPort = 1;
   public static final double manipulatorGamepadSensitivity = 0.6; // Value selected from 2018 code
-  public static final int manipulatorGamepadDeadzone = 0; // Value selected from 2018 code
+  public static final double manipulatorGamepadDeadzone = 0.05; // Value selected from 2018 code
 
   /* Drivetrain */
   public static final double speedMultiplier = .7;  
 
   /* Intake */
   public static final double cargoIntakeInSpeed = 1.0;
-  public static final double cargoIntakeHoldSpeed = 0;
+  public static final double cargoIntakeHoldSpeed = 0.0625;
   public static final double cargoIntakeOutSpeed = -1.0;
 
-  public static final double cargoIntakeCurrentLimit = 20;
+  public static final double cargoIntakeCurrentLimit = 0;
```

Comment:
intake limit code needs to be fixed OR cherry picked out of this branch before it's merged.